### DISCUSSION TITLE
اضافه کردن رنج ایپی دلخواه خودمون د ر نسخه ویندوزی

### DISCUSSION THREAD

**Discussion by armanem3** [MortezaBashsiz/CFScanner#256] [General]
سلام وخسته نباشید خدمت دوستان عزیز امیدوارم وضعیت این نتا بهترشه 
سوال بنده اینه که رنج ایپی دلخواه خودمونو چجوری اضافه کنیم به برنامه نسخه ویندوزی ؟اگر کسی تونسته اینکارو بکنه لطفا به منم راهنمایی بکنه منونم از همه کسانی که برای نت ازاد میجنگن:)

**Comment by goingfine** [comment]
منوی فایل گزینه load custom ip range
